web/src/hash_parser.ts

```diff
@@ -91,6 +91,20 @@ export function is_create_new_stream_narrow(): boolean {
     return window.location.hash === "#streams/new";
 }
 
+// This checks whether the user is in the stream settings menu
+// and is opening the 'Subscribers' tab on the right panel.
+export function is_on_stream_subscribers(): boolean {
```

Comment:
Maybe this function can be named better. Something like `is_subscribers_section_opened_for_stream` would work.

Reply:
Ah ok, I was concerned that it might get a bit long but I suppose being clear is more important. I'll keep that in mind